changelog_unreleased/handlebars/11899.md

```diff
@@ -0,0 +1,23 @@
+#### Support Glimmer's named blocks (#11899 by @duailibe)
+
+Prettier already supported this feature, but it converted empty named blocks to self-closing, which is not supported by the Glimmer compiler.
+
+See: [Glimmer's named blocks](https://emberjs.github.io/rfcs/0460-yieldable-named-blocks.html).
+
+<!-- prettier-ignore -->
+```jsx
```

Comment:
Is there a better language for this? I don't know about this, just a question.

Reply:
oops, that was actually an oversight.. there's  `hbs` I think

Reply:
We can check how we use in the blog post before.